### DISCUSSION TITLE
为什么我的量算组件中的坐标量算在地图上点击完需要挪动下地图才能出来看板数据

### DISCUSSION THREAD

**Discussion by 1510171848** [zouyaoji/vue-cesium#469] [Q&A] [answered]
为什么我的量算组件中的坐标量算在地图上点击完需要挪动下地图才能出来看板数据，清空画布也是需要挪动下地图才能全部清空
![Snipaste_2023-11-14_16-21-02](https://github.com/zouyaoji/vue-cesium/assets/43635418/19f9be22-5e02-467e-a04e-c5cd7156a875)

**Comment by zouyaoji** [comment]
不太清楚了，提供信息无法重现。建议你在线文档操作试试，在线文档也不行说明是你环境的问题，在线文档行，你的代码不行，说明是你代码的问题。

**Comment by 1510171848** [comment, marked as answer]
找到了 vc-viewer   request-render-mode这个属性开启后影响的
